sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -95,30 +101,40 @@ TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder()
 
 credential.update("{new_key}");
 ```
-##### **Option 2**: Create TextAnalyticsClient with Azure Active Directory Credential
-To use an [Azure Active Directory (AAD) token credential][aad_credential],
-provide an instance of the desired credential type obtained from the [azure-identity][azure_identity] library.
-Note that regional endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain] 
-name for your resource in order to use this type of authentication.
+##### Create a Text Analytics client using Microsoft identity platform (formerly Azure Active Directory)
+Azure SDK for Java supports an Azure Identity package, making it easy to get credentials from Microsoft identity
+platform. 
 
 Authentication with AAD requires some initial setup:
-* [Install azure-identity][install_azure_identity]
+* Add the Azure Identity package
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;dependency})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.0.6</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
 * [Register a new AAD application][register_AAD_application]
```

Comment:
We want to continue calling it AAD here?

Reply:
can update it to Azure Active Directory. was followed python readme.